flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -100,6 +100,11 @@
 	public static final String JOB_MANAGER_IPC_PORT_KEY = "jobmanager.rpc.port";
 
 	/**
+	 * The config parameter defining the flink web directory to be used by the webmonitor.
+	 */
+	public static final String WEB_MONITOR_DIRECTORY_KEY = "web.monitor.tmpdir";
```

Comment:
This config key should follow the naming scheme of the other web monitor config keys (see `public static final String JOB_MANAGER_WEB_PORT_KEY = "jobmanager.web.port";`), for example `jobmanager.web.tmpdir`.